pages/trial.tsx

```diff
@@ -0,0 +1,372 @@
+                      </small>
+                    )}
+                  </Form.Group>
+                  <Form.Group>
+                    <Form.Label>Company Website *</Form.Label>
+                    <Form.Control
+                      {...register("companyWebsite", {
+                        required: true,
+                        pattern: urlPattern,
+                      })}
+                      type="text"
+                      name="companyWebsite"
+                      defaultValue=""
+                      onChange={onSubdomainChange}
+                      disabled={disabled}
+                    />
+                  </Form.Group>
+                  {errors.companyWebsite && (
+                    <small style={{ color: "red" }}>
+                      A valid url is required.
+                    </small>
+                  )}
+                  <Form.Group>
+                    <div className="my-3 text-center">
+                      <span>Your Grouparoo subdomain will be: </span>
+                      {editSubdomain ? (
+                        <>
+                          <div className="text-break my-1 h5">
+                            <Form.Control
+                              {...register("subdomain", {
+                                required: true,
+                              })}
+                              type="text"
+                              name="subdomain"
+                              onChange={onSubdomainChange}
+                              defaultValue={subdomain}
+                            />
+                            <span>.grouparoo.app</span>
+                          </div>
+                        </>
+                      ) : (
+                        <div className="text-break my-1 h5">
+                          {" "}
+                          <span>
+                            {subdomain}
+                            <span className="text-nowrap">.grouparoo.app</span>
+                          </span>
+                        </div>
+                      )}
+                      <Error message={error.subdomain} />
+
+                      {errors.subdomain && (
+                        <small style={{ color: "red" }}>
+                          A valid subdomain is required.
+                        </small>
+                      )}
+
+                      <Button variant="link" onClick={onEditSubdomainClick}>
+                        {editSubdomain ? "save" : "change"}
+                      </Button>
+                    </div>
+                  </Form.Group>
+                  <Form.Group>
+                    <div className="my-3">
+                      <Form.Check
+                        {...register("check", { required: true })}
+                        type="checkbox"
+                        id="privacy-policy-check"
+                        defaultValue="false"
+                        label={privacyPolicyLabel}
```

Comment:
Perhaps the label should be centered and in a smaller font?

<img width="1099" alt="Screen Shot 2021-06-15 at 4 08 28 PM" src="https://user-images.githubusercontent.com/303226/122134665-ece35300-cdf3-11eb-95e7-5ed95d730379.png">